internal/pkg/deploy/cloudformation/stack/lb_network_web_service_integration_test.go

```diff
@@ -0,0 +1,141 @@
+//go:build integration || localintegration || temporary
+// +build integration localintegration temporary
+
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package stack_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/aws/copilot-cli/internal/pkg/deploy"
+	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation/stack"
+	"github.com/aws/copilot-cli/internal/pkg/template"
+
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	nlbSvcManifestPath = "svc-nlb-manifest.yml"
+)
+
+func TestNetworkLoadBalancedWebService_Template(t *testing.T) {
+	testCases := map[string]struct {
+		envName       string
+		svcStackPath  string
+		svcParamsPath string
+	}{
+		"test env": {
+			envName:       "test",
+			svcStackPath:  "svc-nlb-test.stack.yml",
+			svcParamsPath: "svc-nlb-test.params.json",
+		},
+		"dev env": {
+			envName:       "dev",
+			svcStackPath:  "svc-nlb-dev.stack.yml",
+			svcParamsPath: "svc-nlb-dev.params.json",
+		},
+		"prod env": {
+			envName:       "prod",
+			svcStackPath:  "svc-nlb-prod.stack.yml",
+			svcParamsPath: "svc-nlb-prod.params.json",
+		},
+	}
+	val, exist := os.LookupEnv("TAG")
+	require.NoError(t, os.Setenv("TAG", "cicdtest"))
```

Comment:
"Setenv" instead of "SetEnv"? Amazing 🧐

Reply:
i knowwww the whole [`os`](https://pkg.go.dev/os) package's function naming feels like a huge "I gave up"